cdist/conf/type/__ufw/man.rst

```diff
@@ -0,0 +1,59 @@
+cdist-type__ufw(7)
+================
```

Comment:
@markasoftware To avoid sphinx warnings (iirc) `=` line length should be equal to text length.

Reply:
Only this little thing.

And can you rebase your branch to have only one commit?
For nicer history :)

Reply:
Sure. I would have liked to split into two commits (one for __ufw and another for __ufw_rule) but didn't bother.

Reply:
Also, you as a maintainer can do this via the GitHub UI when merging a PR: https://help.github.com/en/articles/about-pull-request-merges#squash-and-merge-your-pull-request-commits